utils.js

```diff
@@ -0,0 +1,68 @@
+// TODO
+// await use axios post seedData fn to push data to db
+// done: delete file directory(or add to gitignore so zip files doesn’t push to Github) + drain
+
+const fs = require('fs');
+const path = require('path');
+const unzipper = require('unzipper'); // npm lib used to unzip files
+const { createReadStream } = require('fs');
+
+// Async fs reads and unzip files to a directory
+const folderName = 'Archive.zip';
+const pathToExtract = 'unzippedArchive';
+
+const unzipFolder = (zippedFolder, destination) => {
+  fs.createReadStream(`./${zippedFolder}`).pipe(
+    unzipper.Extract({ path: __dirname + `/${destination}` })
+  );
+};
+
+class File {
+  constructor(filename) {
+    this.filename = filename;
+    this.fileContents;
+  }
+
+  // Read a file and save into the constructor
+  readFile() {
+    return new Promise((resolve, reject) => {
+      try {
+        fs.createReadStream(this.filename)
+
+          .on('data', chunk => {
+            this.fileContents = JSON.parse(chunk.toString());
+          })
+
+          .on('error', err => {
+            reject(err);
+          })
+
+          .on('close', () => {
+            resolve(this.fileContents);
+          });
+      } catch (err) {
```

Comment:
You don't need this the `try..catch` you're already handling errors on line 36. The catch will never hit.